DesktopReact/frontend/src/components/post/Post.js

```diff
@@ -7,11 +7,11 @@ import FooterReply from "../FooterReply/FooterReply";
 
 const Post = ({userName, postBody}) => {
 
-    const [collapse, setCollapse] = useState(true)
-    const Toggle = () => setCollapse(!collapse)
+    const [collapse, setCollapse] = useState(false)
```

Comment:
should this always start un-collapsed?

Reply:
Flipped boolean value in the function, so is inverted in the state